packages/cluster/src/commands/down.js

```diff
@@ -22,23 +25,23 @@ const run = async function({ name, clean, getCache, cluster, ...argv }) {
         process.exit(1)
     }
 
-    console.log(`Winding down cluster ${chalk.cyan(name)}`)
+    reporter.info(`Winding down cluster ${chalk.cyan(name)}`)
     try {
         await exec({
             cmd: 'docker-compose',
             args: [
                 '-p',
-                makeComposeProject(name),
+                name,
                 '-f',
                 path.join(cacheLocation, 'docker-compose.yml'),
                 'down',
             ].concat(clean ? ['--volumes'] : []),
-            env: {
-                DHIS2_CORE_IMAGE: defaults.image,
-                DHIS2_CORE_NAME: name,
-                DHIS2_CORE_PORT: defaults.port,
-            },
+            env: makeEnvironment(cfg),
         })
+
+        if (clean) {
+            cleanCache(argv)
```

Comment:
I think it makes sense to purge the cache for the cluster when we run `--clean`.

Reply:
Could remove this and users could still do it manually with `d2 debug cache purge cluster/{name}` though. Not sure if I expect the configuration to be saved when I run `d2 cluster down {name} --clean` or not..

Reply:
Since we support cached and persistent configurations, I think it makes sense to clear the cache when doing `--clean`.